src/test/java/spoon/test/imports/ImportTest.java

```diff
@@ -1047,7 +1046,7 @@ public void testmportInCu() throws  Exception{
 		assertThat(code, containsString("ArrayList<String> list = new ArrayList<>()"));
 
 		// cleaning
-		output.delete();
+		Files.delete(output);
```

Comment:
Test cleanup after asserts isn't great, this should be in a `finally`. Or better yet we should use a tempfile from junit5.